cclib/parser/xtbparser.py

```diff
@@ -0,0 +1,491 @@
+        #
+        # ================
+        # final structure:
+        # ================
+        # energy: -37.337374171651 gnorm: 0.000298918814 xtb: 6.4.0 (d4b70c2)
+        # xtb     01212213163D
+        # xtb: 6.4.1 (unknown)
+        # 18 19  0     0  0            999 V2000
+        #   -1.0699    0.0056   -0.3196 C   0  0  0  0  0  0  0  0  0  0  0  0
+        #   -0.1726   -0.9763   -0.3757 O   0  0  0  0  0  0  0  0  0  0  0  0
+        #   -0.4790    1.1791    0.0854 C   0  0  0  0  0  0  0  0  0  0  0  0
+        # ...
+        #    1.6615    1.5496    0.6104 H   0  0  0  0  0  0  0  0  0  0  0  0
+        #    5.2547   -2.3860    0.3861 H   0  0  0  0  0  0  0  0  0  0  0  0
+        #    3.3873   -4.1943   -0.4469 H   0  0  0  0  0  0  0  0  0  0  0  0
+        #  1  3  2  0  0  0  0
+        #  2  1  1  0  0  0  0
+        #  2  7  1  0  0  0  0
+        # ...
+        #  14 13  2  0  0  0  0
+        #  15  1  1  0  0  0  0
+        #  18 14  1  0  0  0  0
+        # M  END
+        if 'final structure' == line.strip()[:15]:
+            self.skip_line(inputfile, '=')
+
+            if self.metadata["coord_type"] == 'xyz':
+                atomnos = []
+                atomcoords = []
+                for line in inputfile:
+                    # Ending criteria for xyz is a blank line at the end of the coords block 
+                    if line == ' \n':
+                        break
+                    if line[0].isupper():
+                        atom, x, y, z = line.split()
+                        atomnos.append(self.table.number[atom])
+                        atomcoords.append([float(x), float(y), float(z)])
+                self.set_attribute('natom', len(atomnos))
+                self.set_attribute('atomnos', atomnos)
+                self.set_attribute("atomcoords", atomcoords)
+                #print(lines)
+
+            elif self.metadata["coord_type"] == 'sdf' or self.metadata["coord_type"] == 'mol':
+                atomnos = []
+                atomcoords = []
+                # Ending criteria for sdf\mol is the END at the end of the coord block
+                while 'END' != line.strip()[-3:]:
+                    # Atoms block start with 3 blank spaces, bonds block starts with 1
+                    if line[:3] == '   ':
+                        x, y, z, atom  = line.split()[:4]
+                        atomnos.append(self.table.number[atom])
+                        atomcoords.append([float(x), float(y), float(z)])
+                    line = next(inputfile)
+                self.set_attribute('natom', len(atomnos))
+                self.set_attribute('atomnos', atomnos)
+                self.set_attribute("atomcoords", atomcoords)
+            
+            else:
+                pass
+
+
+        # Get Molecular Orbitals energies and HOMO index
+        # xTB trunctaes the MO list so we need to take care of that.
+        # Unkown energies will be given NaN as a value
```

Comment:
Try running this part of the parser with higher electronic temperature `--etemp 5000` or so.